### PR TITLE
Add Quizzes landing page

### PR DESCRIPTION
**Pull request by michellusthof**
- [x] Add landing page page file
- [x] Add page content
- [x] Create page stylesheet
- [x] Add CTA information
- [x] Add hero image
- [x] Add downloadable pdf
- [x] Add screenshot

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Since I committed on this branch we need another reviewer.

File: source/stylesheets/pages/_quizzes.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    background-position: center;
+
+    h1 {
+      color: $white;
+    }
+  }
+
+  :not(.footer):not(.cta-bottom) {
+    .row {
+      max-width: 1260px;
```
Comment (sn3p) [inline source/stylesheets/pages/_quizzes.scss]: Don't we have/want these to be variables?

Comment (RonaldDijkstra) [reply source/stylesheets/pages/_quizzes.scss]: Good call.

**Comment by RonaldDijkstra** [comment]
Merge and it deploys 😎

**Pull request merged**